packages/kbn-es/src/utils/docker.ts

```diff
@@ -310,12 +310,7 @@ export function resolveDockerImage({
  */
 export function resolvePort(options: ServerlessOptions | DockerOptions) {
   if (options.port) {
-    return [
-      '-p',
-      `127.0.0.1:${options.port}:${options.port}`,
-      '--env',
-      `http.port=${options.port}`,
-    ];
+    return ['-p', `0.0.0.0:${options.port}:${options.port}`, '--env', `http.port=${options.port}`];
```

Comment:
because we're running an Ubuntu VM in Multipass/Vagrant we need to be able to reach out to the ES docker instance, and as far as I know, it's the simplest way to achieve it, please let me know what you think @elastic/kibana-operations

Reply:
It's simplest but unfortunately docker port forwarding tends to rewrite firewall rules, and we don't want our CI agents with accessible services over TCP.

https://docs.docker.com/network/#published-ports

I would recommend creating a private network in your VM configuration and binding elasticsearch to that network.  We can update this to take a full IP/port combination as an argument .

Reply:
@jbudz please let me know if the updated solution works for you

Reply:
Is it possible to put this behind a flag?  I think by default we'll want it off for developers.

Reply:
Second question (sorry I know we discussed earlier, I've thought on it more since) - how much effort extra effort is it to create and use a  private network?  While less open, binding to an existing LAN isn't ideal for developers on public networks.

Reply:
Resolved with https://github.com/elastic/kibana/pull/167720/commits/be6b0e58616d627eae7140ebfaef5ff28c5ec410